internal/test/wmcb/templates/cni.j2

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Should we call this file as j2? j2 is specifically for python templating.

Reply:
I am open to changing the name. Please provide a suggestion.